lib/sentry/logger_backend.ex

```diff
@@ -42,6 +42,10 @@ defmodule Sentry.LoggerBackend do
   to `[]`. If set to `:all`, all metadata will be included. `:all` is available
   since v9.0.0 of this library.
 
+  * `:tags` - To include Logger metadata as tags in reports, the `:tags` key can be set to a list of keys.
```

Comment:
Should we call this `:tags_from_metadata`?

Reply:
Also, sorry just seeing now that this is `Sentry.LoggerBackend`: let's not add this option there and make it only available for the Logger handler. We will eventually deprecate and then remove the Logger backend, so better to not add new features to it.